weaviate/collection/classes/config.py

```diff
@@ -1294,14 +1427,22 @@ def inverted_index(
 
     @classmethod
     def multi_tenancy(cls, enabled: bool = False) -> _MultiTenancyConfigCreate:
+        """Create a `MultiTenancyConfigCreate` object to be used when defining the multi-tenancy configuration of Weaviate.
+
+        Argumentss:
+                enabled: Whether multi-tenancy is enabled. Defaults to `False`.
+
+        Returns:
+            A `MultiTenancyConfigCreate` object.
```

Comment:
This return is probably not helpful (same for the other internal classes that are returned)

Reply:
Okay cool, it was annoyingly verbose but I stuck to a standard. Will remove everywhere!